src/Worker/Core/Shims/TaskEntityShim.cs

```diff
@@ -68,6 +72,7 @@ public override async Task<EntityBatchResult> ExecuteOperationBatchAsync(EntityB
             }
             catch (Exception applicationException)
             {
+                this.logger.OperationError(applicationException, this.entityId, current.Operation!);
```

Comment:
what does the `!` operator do at the end of this expression? My guess is something related to nullability

Reply:
Yeah, it suppresses a nullability warning, saying "I know this isn't null here"